platform/ios/src/MGLMapView.h

```diff
@@ -526,6 +526,13 @@ MGL_EXPORT IB_DESIGNABLE
  */
 @property(nonatomic, getter=isHapticFeedbackEnabled) BOOL hapticFeedbackEnabled;
 
+/**
+ A Boolean value that determines whether the updating pitch will also affect the altitude.
+ 
+ When this property is set to `YES`, pitch will work independently from altitude.
+ */
+@property(nonatomic, getter=isAltitudeUnaffectedByPitch) BOOL altitudeUnaffectedByPitch;
```

Comment:
Is it OK if we specify that this property controls the camera updates and nothing more, something along the lines —feel free to add different wording—:

> A Boolean value that determines wether updating the camera pitch will also affect the altitude.
When this property is set to `YES`, pitch will work independently from altitude.
The default value of this property is `NO`.

Also I think the documentation needs a way to clarify the scope of this change and how people may take advantage, I know you already have a repo with experimental changes, but communicating how devs may take advantage of this I think is important.

In the SDK we follow the convention of avoid using negatives as part of boolean properties. As @julianrex said `isAltitudeAffectedByPitch` follows that pattern. But also is it more clear the intent using something like `isCameraAltitudeAffectedByPitch`?